docs/user_guide.md

```diff
@@ -252,6 +256,97 @@ The images can be filtered by labels:
+1. Click **Add** at the bottom of the window to add the list.
+1. Optionally uncheck the **Never expires** checkbox and use the calendar selector to set an expiry date for the whitelist.
+   ![Add system CVEs](img/cve-whitelist3.png)
+1. Click **Save** at the bottom of the page to save your settings.
+   
+After you have created a system whitelist, you can remove CVE IDs from the list by clicking the delete button next to it in the list. You can click **Add** to add more CVE IDs to the system whitelist.
+
+![Add and remove system CVEs](img/cve-whitelist4.png)
+
+### Configure a Per-Project CVE Whitelist
+
+By default, the system whitelist is applied to all projects. You can configure different CVE whitelists for individual projects, that override the system whitelist. 
+
+1. Go to **Projects**, select a project, and select **Configuration**.
+1. Under **CVE whitelist**, select **Project whitelist**. 
+   ![Project CVE whitelist](img/cve-whitelist5.png)
+1. Optionally click **Add System** to add all of the CVE IDs from the system CVE whitelist to this project whitelist.
+1. Click **Add** and enter a list of additional CVE IDs to ignore during vulnerability scanning of this project. 
+   ![Add project CVEs](img/cve-whitelist6.png)
+
+   Either use a comma-separated list or newlines to add multiple CVE IDs to the list.
+1. Click **Add** at the bottom of the window to add the CVEs to the project whitelist.
+1. Optionally uncheck the **Never expires** checkbox and use the calendar selector to set an expiry date for the whitelist.
+1. Click **Save** at the bottom of the page to save your settings.
+
+After you have created a project whitelist, you can remove CVE IDs from the list by clicking the delete button next to it in the list. You can click **Add** at any time to add more CVE IDs to the whitelist for this project. 
+
+If CVEs are added to the system whitelist after you have created a project whitelist, click **Add System** to add the new entries from the system whitelist to the project whitelist. 
+
+**NOTE**: If CVEs are deleted from the system whitelist after you have created a project whitelist, and if you added the system whitelist to the project whitelist, you must manually remove the deleted CVEs from the project whitelist. If you click **Add System** after CVEs have been deleted from the system whitelist, the deleted CVEs are not automatically removed from the project whitelist.
+
+## Set Project Quotas
+
+To exercise control over resource use, as a system administrator you can set  quotas on projects. You can limit the number of tags that a project can contain and limit the amount of storage capacity that a project can consume. You can set default quotas that apply to all projects globally. You can also set quotas on individual projects. If you set global default quotas and also set different quotas on individual projects, the per-project quotas override the global default quotas.

```

Comment:
`You can set default quotas that apply to all projects globally.` there is something not clearly for this.

The default quotas only have effect for the new created projects, once the project created, its quotas will be confirmed. The Changing of the default quota settings will not have effect for the projects already exist.

As the above description, there is no scenario to mean `If you set global default quotas and also set different quotas on individual projects, the per-project quotas override the global default quotas.`,  set different quotas on individual projects is just update quotas to the individual projects.

Reply:
Thanks @heww I reworded this in https://github.com/goharbor/harbor/pull/8869/commits/bb795ef05621fb35aa482e19187476799501e57c. Is this OK now?